src/Http/Controllers/CommentController.php

```diff
@@ -34,28 +32,8 @@ public function createOrUpdateTaskComment(Request $request)
         }
     }
 
-    public function deleteTaskComment($id)
+    public function deleteTaskComment(Request $request)
     {
-        try {
-            $comment = Comment::find($id);
-
-            Log::createLog(
-                Auth::user()->id,
-                Log::TYPE_COMMENT_DELETED,
-                $comment->comment,
-                null,
-                $comment->id,
-                'Xguard\LaravelKanban\Models\Comment'
-            );
-
-            $comment->delete();
-
-        } catch (\Exception $e) {
-            return response([
-                'success' => 'false',
-                'message' => $e->getMessage(),
-            ], 400);
-        }
-        return response(['success' => 'true'], 200);
+        return (new DeleteTaskCommentAction(['comment_data' => $request->all()]))->run();
     }
```

Comment:
app(DeleteTaskCommentAction::class)->fill(['comment_data' => $request->all()]))->run() and test it works, you can then use prophecies to mock it in the container